beta/README.md

```diff
@@ -29,20 +29,28 @@ The samples demonstrate the usage of compression algorithms for two different us
 
 ## Usage
 The NNCF TF is organized as a regular Python package that can be imported in an arbitrary training script.
-The basic workflow is loading a JSON configuration script containing NNCF-specific parameters determining the compression to be applied to your model, and then passing your model along with the configuration script to the `nncf.create_compressed_model` function.
+The basic workflow is loading a JSON configuration script containing NNCF-specific parameters determining the compression to be applied to your model, and then passing your model along with the configuration script to the `nncf.tensorflow.create_compressed_model` function.
```

Comment:
According to the code below, it should be `beta.nncf.tensorflow.create_compressed_model`

Reply:
Thanks, you fixed it:)